homework1/task05.py

```diff
@@ -0,0 +1,34 @@
+"""
+Given a list of integers numbers "nums".
+
+You need to find a sub-array with length less equal to "k", with maximal sum.
+The written function should return the sum of this sub-array.
+Examples:
+    nums = [1, 3, -1, -3, 5, 3, 6, 7], k = 3
+    result = 16
+"""
+from typing import List
+
+
+def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
+    """Find largest subarray sum with at-least k elements in it.
+
+    Args:
+        nums: list of integers.
+        k: number of elements of subarray.
+    Returns:
+        maximum sum of a subarray with at-least k elements.
+    """
+    if len(nums) == 0 or k <= 0 or k > len(nums):
+        raise ValueError("k should not be greater then the size of list of integers")
+    summ = 0
+    for i in range(k):
```

Comment:
Do you know how to replace the lines 24-26 with just one line using `sum()` function and list slicing?